python/repositories.bzl

```diff
@@ -35,6 +35,13 @@ def py_repositories():
 # Remaining content of the file is only used to support toolchains.
 ########
 
+# Parse the bazel version string from `native.bazel_version`.
+# e.g.
+# "0.10.0rc1 abc123d" => (0, 10, 0)
+# "0.3.0" => (0, 3, 0)
+def _parse_native_bazel_version(bazel_version):
+    return tuple([int(n) for n in bazel_version.split(".")])
```

Comment:
I took this from https://github.com/bazelbuild/bazel-skylib/blob/main/lib/versions.bzl#L22 but I don't really want this file to take a dependency on skylib for such simple functionality. 🤷